docs/source/markdown/podman-search.1.md

```diff
@@ -65,6 +68,10 @@ Example if limit is 10 and two registries are being searched, the total
 number of results will be 20, 10 from each (if there are at least 10 matches in each).
 The order of the search results is the order in which the API endpoint returns the results.
 
+**--list-tags**
+
+Search tags available in the repository. Note: the result only contains the Image name and its tag.
```

Comment:
As we can use search on `registry.redhat.io/` and get all of the images in the repository:

```suggestion
List the available tags in the repository for each image found. Note: the result contains the Image name and its tag, one line for every tag associated with the image.
```